router/image/templates/nginx.conf

```diff
@@ -264,10 +264,6 @@ http {
 
             proxy_pass                  http://{{ Base $service.Key }};
         }
-        {{ else }}
```

Comment:
what I meant is that this should not have been removed. We still need this code block. No modification is required here. Sorry about the confusion